iothub/device/samples/PnpDeviceSamples/readme.md

```diff
@@ -0,0 +1,17 @@
+### Plug and Play Samples
+
+These samples demonstrate how a plug and play enabled device interacts with IoT hub, to:
+- Send telemetry.
+- Update read-only and read-write porperties.
+- Respond to command invocation. 
+
+The samples demonstrate two scenarios:
+- The device is modeled as a plug and play device, having only a root interface - [Thermostat][d-thermostat]
+  - This model defines root level telemetry, read-only and read-write properties and commands.
+- The device is modeled as a plug and play device having multiple components - [Temperature controller][d-temperature-controller].
+  - This model defines root level telemetry, read-only property and commands.
+  - It also defines two [Thermostat][d-thermostat] components, and a [device information][d-device-info] component.
```

Comment:
no need to add a link to thermostat sample. If we really want to show component then I would add a link to dtdl showing this component as part of the model

Reply:
the thermostat sample has the model json file

Reply:
sure but that doesn't help in this comment

Reply:
I can add a link to the json file in the thermostat sample, sure